src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesCore.cs

```diff
@@ -15,8 +15,9 @@ namespace BizHawk.Emulation.Cores.Nintendo.BSNES
 	[ServiceNotApplicable(new[] { typeof(IDriveLight) })]
 	public partial class BsnesCore : IEmulator, IDebuggable, IVideoProvider, ISaveRam, IStatable, IInputPollable, IRegionable, ISettable<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings>, IBSNESForGfxDebugger, IBoardInfo
 	{
+		[CoreConstructor(VSystemID.Raw.GB)]
+		[CoreConstructor(VSystemID.Raw.GBC)]
```

Comment:
This is just incorrect. BSNES doesn't magically emulate a new CPU variant by just adding an attribute.

This is my main concern with this PR, a needless weakening of the "system" semantics that are already too weak.

Reply:
I have changed it so that the default system id is still SGB when the core is BSNES and now the BSNES classes don't need any changes.

Reply:
That's not what I meant, but it could work.